### DISCUSSION TITLE
MBO

### DISCUSSION THREAD

**Discussion by Niyasmkkv** [iptv-org/community#698] [Q&A]
**Channel Name:** MBO
**Channel ID:** MBO.in
**Country:** IN
**Language:** Malayalam
**Website:** blob:https://mhdtvworld.xyz/c95fb22a-2761-4c17-b7bc-a60a6bba091e
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by Sphinxroot** [comment]
https://mhdtvworld.xyz/delta/planet/max.php?id=4065365&e=.m3u8

cant play it in kodi

**Comment by Niyasmkkv** [comment]
Apparently this website is blocked in India, If the channels are from this server, it is useless for Indians.

**Comment by BellezaEmporium** [comment]
> 
> 
> https://mhdtvworld.xyz/delta/planet/max.php?id=4065365&e=.m3u8
> 
> cant play it in kodi

Referrer locked, that might be why.

Try with referrer https://mhdtvworld.xyz/

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19715
